assets/src/data/model/event/test/index.js

```diff
@@ -24,7 +26,10 @@ describe( 'mapOrderBy()', () => {
 
 describe( 'whereConditions()', () => {
 	it( 'returns expected default for empty object passed in', () => {
-		expect( whereConditions( {} ) ).toEqual( '' );
+		expect( whereConditions( {} ) ).toEqual(
+			'where[Datetime.DTT_EVT_end**expired][]=>' +
```

Comment:
This will need fixed to be `GREATER_AND_EQUAL` instead of `=>` if the same fix is done in the method being tested.